x/config.go

```diff
@@ -35,7 +35,7 @@ type Options struct {
 	//
 	// query-edge uint64 - maximum number of edges that can be returned in a query
 	// normalize-node int - maximum number of nodes that can be returned in a query that uses the
-	//                      normalize directive
+	//                      normalized directive
```

Comment:
normalize is a directive in dql I believe.

Reply:
okay